pkg/kube/client.go

```diff
@@ -87,51 +131,124 @@ type Client interface {
 	NewPortForwarder(podName string, ns string, localAddress string, localPort int, podPort int) (PortForwarder, error)
 }
 
-var _ Client = &client{}
-
 // Client is a helper wrapper around the Kube RESTClient for istioctl -> Pilot/Envoy/Mesh related things
 type client struct {
-	*kubernetes.Clientset
+	kubernetes.Interface
 	*rest.RESTClient
+
+	kubeInformer informers.SharedInformerFactory
```

Comment:
nit: any reason for the grouping?

Reply:
Not really, just copy+pasted from my old struct which was formatted this way. What do you prefer? Not blank lines or keeping informers together?